website_docs/getting-started.md

```diff
@@ -43,6 +43,8 @@ from opentelemetry.sdk.trace.export import (
 )
 
 provider = TracerProvider()
+# Note: Using SimpleSpanProcessor is strongly discouraged in production-grade
+# applications. Please use BatchSpanProcessor instead.
```

Comment:
Why do we recommend SimpleSpanProcessor at all? It feels strange to guide users one way and then tell them not to do it. I suppose the only reason is that simple processor provides more immediate feedback and that can be helpful when someone is getting started but I don't think it is that beneficial.

Reply:
I thought this simple script exits before writing trace to stdout if `BatchSpanProcessor` is used. Updated.